spec/spec_helper.rb

```diff
@@ -1,4 +1,5 @@
 require 'coveralls'
+require 'email_spec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.